examples/gum/symbol_util/src/main.rs

```diff
@@ -0,0 +1,18 @@
+use frida_gum::symbol_details_from_address;
+use frida_gum::{Gum, Module};
+
+fn main() {
+    let _gum: Gum = unsafe { Gum::obtain() };
```

Comment:
This needs to go in a `lazy_static` block and be declared as a global, otherwise it will drop before the scope ends.